apps/dashboard/src/components/workflow-editor/steps/digest/period.tsx

```diff
@@ -8,7 +8,8 @@ const PERIOD_OPTIONS = [
   { value: PeriodValues.DAY, label: 'day' },
   { value: PeriodValues.WEEK, label: 'week' },
   { value: PeriodValues.MONTH, label: 'month' },
-  { value: PeriodValues.YEAR, label: 'year' },
+  // disable the year option for now as it might affect Redis memory and performance
+  // { value: PeriodValues.YEAR, label: 'year' },
```

Comment:
I think we can remove it entirely. There's no need to keep those lines for now.
we can revisit and add support when needed.

Reply:
if we want to remove it, there is more code... but it might be hard to bring it back after some time, so maybe let's leave it for now